### PR TITLE
sample code to create datastore using disk uuid

### PR DESCRIPTION
**Pull request by arvindayyangar**
I wasted a lot of time trying to get this code working, I am sure this sample code will help others. I posted question on how to use QueryVmfsDatastoreCreateOptions and other API's in various forums but got no response.

### CODE REVIEW

File: samples/esx_create_ds.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            hostdssystem = esx_host.configManager.datastoreSystem
+            dssystem = vim.host.DatastoreSystem
+            vmfs_ds_options = dssystem.QueryVmfsDatastoreCreateOptions(
+                                                       hostdssystem, dp, 5)
+            if dsname is not None:
+                vmfs_ds_options[0].spec.vmfs.volumeName = dsname
+
+            new_ds = dssystem.CreateVmfsDatastore(hostdssystem,
+                                                  vmfs_ds_options[0].spec)
+        except vim.fault.NotFound:
```
Comment (tianhao64) [inline samples/esx_create_ds.py]: no need to handle exception.

File: samples/esx_create_ds.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            print "Unexpected error:", e
+        else:
+            print "Datastore created"
+
+
+def main():
+    args = get_args()
+
+    # connect this thing
+    si = connect.SmartConnect(
```
Comment (tianhao64) [inline samples/esx_create_ds.py]: please make sure the code works in python2.7 and 3.5. 
to bypass server cert verification, pass following to SmartConnect:

sslContext = None
if hasattr(ssl, '_create_unverified_context'):
     sslContext = ssl._create_unverified_context()

Comment (kphatak) [inline samples/esx_create_ds.py]: can you also provide info on how to get list of current UUIDs if this needs to be done dynamically ?

Comment (chintanckg) [inline samples/esx_create_ds.py]: My question is on Removing Datastores using pyvmomi: 

I am writing a script that mounts a datastore and then unmounts it again. 

While mounting I am giving the datastore a name eg. "test". 

But on vCenter, its getting created with the name as UUID. So, when I say RemoveDatastore fand pass "test", it says no such datastore found. 

Please help!

**Comment by pgbidkar** [comment]
Closing the PR due to no updates from long time.
Please create new PR is this sample is required in list of samples here

Thanks

**Pull request closed without merging**